test/pages/HomePage.ts

```diff
@@ -0,0 +1,30 @@
+import { Locator, Page } from "@playwright/test";
+import { base_url } from "../helper-config";
+
+export class HomePage {
+  page: Page;
+  loginLinkLocator: Locator;
+  signupLinkLocator: Locator;
+  welcomeLocator: Locator;
+
+  constructor({ page }: { page: Page }) {
+    this.page = page;
+    this.loginLinkLocator = page.getByRole("link", { name: "Log In" });
+    this.signupLinkLocator = page.getByRole("link", { name: "Sign Up" });
+    this.welcomeLocator = page.getByRole("heading", {
+      name: "Welcome to Open Library",
+    });
+  }
+
+  async navigateToHome() {
```

Comment:
```suggestion
  async navigateToHomePage() {
```
Similar for functions below